### PR TITLE
Features/chat bot/UI improvement

### PR DESCRIPTION
**Pull request by immangat**

### CODE REVIEW

File: lib/core/config/assets/app_icons.dart
Comment (noufilsaqib) [inline lib/core/config/assets/app_icons.dart:1]: you added the forbidden icon to bold, but i dont see it in the bold folder.

File: lib/presentation/chatbot/bloc/chatbot_cubit.dart
Comment (noufilsaqib) [inline lib/presentation/chatbot/bloc/chatbot_cubit.dart:1]: import the failures.dart file using relative path instead

File: lib/presentation/chatbot/bloc/chatbot_state.dart
Comment (noufilsaqib) [inline lib/presentation/chatbot/bloc/chatbot_state.dart:1]: replace period with question mark for the text property

File: lib/presentation/chatbot/pages/chatbot.dart
Comment (noufilsaqib) [inline lib/presentation/chatbot/pages/chatbot.dart:1]: import the new_chat.dart file using relative path instead

File: lib/presentation/chatbot/widgets/chatbot_body.dart
Comment (noufilsaqib) [inline lib/presentation/chatbot/widgets/chatbot_body.dart:1]: import the selected_images.dart file using relative path instead

File: lib/presentation/diagnostics/pages/diagnostics.dart
Comment (noufilsaqib) [inline lib/presentation/diagnostics/pages/diagnostics.dart:1]: import the chatbot_cubit.dart file using relative path instead

File: lib/core/config/assets/app_icons.dart
Comment (noufilsaqib) [inline lib/core/config/assets/app_icons.dart:1]: you added the forbidden icon to bold, but i dont see it in the bold folder

File: lib/presentation/chatbot/bloc/chatbot_cubit.dart
Comment (noufilsaqib) [inline lib/presentation/chatbot/bloc/chatbot_cubit.dart:1]: import the failures.dart file using relative path instead

**Review by noufilsaqib — CHANGES_REQUESTED** [review]
check out the comments

File: lib/core/config/assets/app_icons.dart
Comment (immangat) [reply lib/core/config/assets/app_icons.dart:1]: I deleted it i think because i don't use it anymore

**Review by noufilsaqib — APPROVED** [review]

**Pull request merged**